packages/ketcher-core/src/domain/serializers/ket/ketSerializer.ts

```diff
@@ -234,7 +234,34 @@ export class KetSerializer implements Serializer<Struct> {
         this.validateConnectionTypeAndEndpoints(connection, editor);
       },
     );
-    return { error, parsedFileContent };
+    return {
+      error,
+      parsedFileContent,
+    };
+  }
+
+  convertCoordinatesAndGetMinMax(parsedFileContent: IKetMacromoleculesContent) {
```

Comment:
My proposal is to move it outside serializer. Serializer doesn't know anything about the canvas. Imagine you're sending these coordinates to indigo (backend), what is the size of canvas on backend?
So I believe we need to parse the structure from Indigo and then separately center it on our canvas.